staticfiles/js/authentication/controllers/register.controller.js

```diff
@@ -33,10 +33,11 @@
         */
         function register() {
           cfpLoadingBar.start();
+	// vm.phone and vm.country have to be added here by the one working on the backend( change in models.py )
           Authentication.register(vm.email, vm.firstname, vm.lastname,
             vm.password1, vm.password2).then(function () {
-              
-              $location.url('/login');
+              console.log("Done")
```

Comment:
it will redirect to the resiterstep2 html page instead of the login page.
Let me know if i am missing something here. Thanks :)